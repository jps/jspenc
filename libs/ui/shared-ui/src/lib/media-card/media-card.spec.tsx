import { render } from '@testing-library/react';

import MediaCard from './media-card';

describe('MediaCard', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<MediaCard href='#' headline='Test Card' children='lorem ipsum.' />);
    expect(baseElement).toBeTruthy();
  });
});
