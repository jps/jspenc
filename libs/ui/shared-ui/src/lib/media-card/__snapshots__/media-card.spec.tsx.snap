// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaCard should render successfully 1`] = `
<body>
  <div>
    <div>
      <div
        class="css-1m0fc5"
      >
        <div
          class="css-1shocxe"
        >
          <div
            class="css-u3mic"
          >
            <div
              class="css-x5nivz"
            >
              <div
                class="css-n925f3"
              >
                <a
                  class="nk-card-link css-12rz4zb"
                  href="/#"
                  target="_blank"
                >
                  <section
                    class="css-kqlwu2"
                  >
                    <h3
                      class="nk-headline-heading css-1qyizy8"
                    >
                      Test Card
                    </h3>
                  </section>
                </a>
              </div>
              <div
                class="css-n925f3"
              >
                <p
                  class="css-1as828z"
                >
                  lorem ipsum.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="nk-layer-organizer"
  />
</body>
`;
